server/src/main/java/org/opensearch/index/fielddata/fieldcomparator/BytesRefFieldComparatorSource.java

```diff
@@ -122,11 +123,6 @@ protected SortedDocValues getSortedDocValues(LeafReaderContext context, String f
                     }
                 }
 
-                @Override
-                public void setScorer(Scorable scorer) {
-                    BytesRefFieldComparatorSource.this.setScorer(scorer);
-                }
```

Comment:
not sure what was the use of this and when we are removing it will this impact anything.

Reply:
It doesn't impact anything since the `setScorer` method is a noop